README.md

```diff
@@ -1,43 +1,152 @@
+nextflow run labsyspharm/mcmicro-nf --in path/to/exemplar-001 --illum
 ```
 
 ### O2 execution
 
+To run the pipeline on O2, two additional steps are required: 1) you must load the necessary O2 modules, and 2) all pipeline calls need to have the flag `-profile O2`.
+
 ``` bash
-# Load necessary modules
+# Load necessary modules (matlab is optional, if not working with TMA)
 module load java matlab conda2
 
 # Get the latest version of the pipeline
 nextflow pull labsyspharm/mcmicro-nf
 
-# Both exemplars are in /n/groups/lsp/cycif/exemplars
-# Individual tools are already pre-installed to /n/groups/lsp/mcmicro
+# All previous commands require an additional `-profile O2` flag
+nextflow run labsyspharm/mcmicro-nf --in path/to/exemplar-001 -profile O2
+nextflow run labsyspharm/mcmicro-nf --in path/to/exemplar-002 --TMA -profile O2
+```
+
+## Handling intermediate files
+
+By default Nextflow writes intermediate files to a `work/` directory inside whatever location you initiate a pipeline run from. The intermediate files allow you to restart a pipeline partway, without re-running everything from scratch. For example, consider the following scenario on O2:
+
+``` bash
+module load java conda2      # <--- OOPS, forgot matlab
+
+# This run will fail with "matlab: command not found"
+nextflow run labsyspharm/mcmicro-nf --in path/to/exemplar-002 --TMA -profile O2
+
+# N E X T F L O W  ~  version 20.01.0
+# Launching `labsyspharm/mcmicro-nf` [backstabbing_fermi] - revision: e5ff35c351 [master]
+# executor >  slurm (2)
+# [-        ] process > illumination   -
+# [57/1c3712] process > ashlar         [100%] 1 of 1 ✔
+# [cf/7b42eb] process > dearray        [100%] 1 of 1, failed: 1 ✘
+# [-        ] process > unmicst        -
+# [-        ] process > s3seg          -
+# [-        ] process > quantification -
+
+# Address the issue by loading the appropriate module
+module load matlab
+
+# Restart the pipeline from the dearray step using `-resume`
+nextflow run labsyspharm/mcmicro-nf --in path/to/exemplar-002 --TMA -profile O2 -resume
+
+# N E X T F L O W  ~  version 20.01.0
+# Launching `labsyspharm/mcmicro-nf` [condescending_wing] - revision: e5ff35c351 [master]
+# executor >  slurm (13)
+# [-        ] process > illumination   -
+# [57/1c3712] process > ashlar         [100%] 1 of 1, cached: 1 ✔      <- NOTE: cached
+# [dd/1928b1] process > dearray        [100%] 1 of 1 ✔
+# [1c/82bcd4] process > unmicst        [100%] 4 of 4 ✔
+# [f7/02146c] process > s3seg          [100%] 4 of 4 ✔
+# [14/25a33c] process > quantification [100%] 4 of 4 ✔
+```
+
+As you run the pipeline on your datasets, the size of the `work/` directory can grow substantially. Two Nextflow features can greatly assist with managing its content. First, you can control where the `work/` directory gets create. On O2, it is recommended to use `/n/scratch2`:
+
+``` bash
+nextflow run labsyspharm/mcmicro-nf --in /path/to/exemplar-001 -profile O2 -w /n/scratch2/eCommonsID/work/
```

Comment:
Suggest `$USER` instead of `eCommonsID` so it can be directly copy-pasted. Let the user's shell replace it.